snuba/datasets/entities/metrics.py

```diff
@@ -161,6 +169,17 @@ def __init__(self) -> None:
         )
 
 
+class OrgMetricsCountersEntity(MetricsEntity):
+    def __init__(self) -> None:
+        super().__init__(
+            writable_storage_key=None,
+            readable_storage_key=StorageKey.METRICS_COUNTERS,
+            value_schema=[],
```

Comment:
I set `value_schema` and `mappers` to be empty since we don't need to do complex sums or aggregations here. This seems to work fine in the tests.

Reply:
I think I would keep those at parity. It's just nice to be able to write `sum(value)` in a SnQL query vs `sumMerge(value)` though both will work